install/minikube-installation-guide.md

```diff
@@ -0,0 +1,240 @@
+
+```sh
+$ git clone https://github.com/kata-containers/packaging.git
+$ cd packaging/kata-deploy
+$ kubectl apply -f kata-rbac.yaml
+$ kubectl apply -f kata-deploy.yaml
+```
+
+This installs the Kata Containers components into `/opt/kata` inside the Minikube node. It can take
+a few minutes for the operation to complete. You can check the installation has worked by checking
+the status of the `kata-deploy` pod. This can be accomplished by running the following:
+
+```sh
+$ podname=$(kubectl -n kube-system get pods -o=name | fgrep kata-deploy | sed 's?pod/??')
+# If 'sleep infinity' is running, the pod has completed its work.
+$ kubectl -n kube-system exec ${podname} -- ps -ef | fgrep infinity
+root        49     1  0 10:55 ?        00:00:00 sleep infinity
+```
+
+> *NOTE:* This check only works for single node clusters, which is the default for Minikube.
+> For multi-node clusters, the check would need to be adapted to check `kata-deploy` had
+> completed on all nodes.
+
+## Enabling Kata Containers
+
+> **Note:** This step is only required for Minikube/Kubernetes versions <= 1.13. Since version
+> v1.14, the `RuntimeClass` is enabled by default. Performing this step on Kubernetes > v1.14 is
+> however benign.
+
+Now the Kata Containers components are installed in the Minikube node, you need to configure
+Kubernetes `RuntimeClass` to know when to use Kata Containers to run a pod.
+
+```sh
+$ curl https://raw.githubusercontent.com/kubernetes/node-api/master/manifests/runtimeclass_crd.yaml > runtimeclass_crd.yaml
+$ kubectl apply -f runtimeclass_crd.yaml
+```
+
+### Register the runtime
+
+Now register the `kata qemu` runtime with that class:
+
+```sh
+$ cd packaging/kata-deploy/k8s-1.14
+$ kubectl apply -f kata-qemu-runtimeClass.yaml
+runtimeclass.node.k8s.io/kata created
+runtimeclass.node.k8s.io/kata-qemu created
+```
+
+Kata Containers should now be installed and enabled in the Minikube cluster.
+
+## Testing Kata Containers
+
+Launch a container that is defined to run on Kata Containers. The enabling is configured by
+the following lines in the yaml file:
+
+```yaml
+    spec:
+      runtimeClassName: kata-qemu
+```
+
+Perform the following action to launch a Kata Containers based pod:
+
+```sh
+$ cd packaging/kata-deploy/examples
+$ kubectl apply -f test-deploy-kata-qemu.yaml
+```
+
+An Apache PHP pod should have been deployed with the Kata Containers runtime. Wait a few moments
+to check it is running:
+
+```sh
+$ kubectl get pods
+NAME                                   READY   STATUS    RESTARTS   AGE
+php-apache-kata-qemu-bc4c55994-p6f6k   1/1     Running   0          1m
+```
+
+There are a couple of ways to verify it is running with Kata Containers. Nominally, it
+should be hard to tell - the idea of Kata Containers is that your pod will run inside a VM,
+but look and feel just like it would as a normal software pod. In
+theory, you should not be able to tell your pod is running as a Kata Containers container.
```

Comment:
These two sentences are essentially saying the same thing. I'd consider dropping the "Nominally, " sentence.

Reply:
done